### PR TITLE
Wmwragg/button updates

### LINKED ISSUE #1752: General design: implement new buttons
General design (was Chat inline link 20a screen): implement all new buttons as per design  

* Design: all assets ready

### PR DESCRIPTION
**Pull request by wmwragg**
This should close #1752. A companion branch matrix-org/matrix-react-sdk:wmwragg/button-updates also needs to be merged.

Signed-off-by: William Wragg wm.wragg@gmail.com

### CODE REVIEW

File: src/components/structures/BottomLeftMenu.js [hunk 1]
```diff
@@ -52,13 +52,13 @@ module.exports = React.createClass({
             <div className="mx_BottomLeftMenu">
                 <div className="mx_BottomLeftMenu_options">
                     <div className="mx_BottomLeftMenu_createRoom" title="Start chat" onClick={ this.onCreateRoomClick }>
-                        <TintableSvg src="img/icons-create-room.svg" width="24" height="24"/>
+                        <TintableSvg src="img/icons-create-room.svg" width="25" height="25"/>
```
Comment (ara4n) [inline src/components/structures/BottomLeftMenu.js]: what is the rationale for increasing these to 25x25?  The original artwork is 24x24, so scaling it up by 4% is going to turn all of the 1px-wide strokes into fuzzyish 1.04px strokes, especially on non-retina displays...

Comment (wmwragg) [inline src/components/structures/BottomLeftMenu.js]: All the new artwork that has been changed are SVGs, and were drawn with 25x25px in mind.

Comment (ara4n) [inline src/components/structures/BottomLeftMenu.js]: ah, cool; hadn't realised the original format had switched to 25x25. all good :)

**Comment by ara4n** [comment]
Sorry for the delay in doing a proper review - have checked out the right branches and taken a look now.

In general, seems great, but there are a few weirdnesses:

Chrome:
image

Zeplin:
image

Obviously the Voip/Video buttons are missing - I suggest in the short term we add a skinned video button until combining both functions into a single button can be done.

However, the weighting of the strokes in the zoom and the file upload button is also wrong relative to the design, as is the vertical centering of the members-panel icon in the top right.  

Happy to merge once the above is fixed; we'll also need to kill the crappy 'leave' button asap in favour of the RoomTile contextual menu, but that's not a hard blocker.

**Comment by ara4n** [comment]
(the room settings icon is right in your design though; i screenshotted the wrong zeplin)

**Comment by ara4n** [comment]
the weighting of the membership icon is also wrong.

**Comment by ara4n** [comment]
This is almost great, other than:
* [ ] the vertical offset of the membership list button doesn't match the design (see below)
* [ ] we are missing the buttons for voip/video. up to you whether we should merge it without and create another ticket, or wait until the assets are available.

screen shot 2016-07-24 at 19 56 53

**Comment by wmwragg** [comment]
I'd say we merge without the assets, and make a new ticket to implement the underlying functionality for the single button, along with the design work

**Pull request merged**